tests/phpunit/integration/Core/Permissions/PermissionsTest.php

```diff
@@ -587,4 +599,71 @@ public function test_view_splash__admin() {
 		$this->dismissed_items->add( 'shared_dashboard_splash' );
 		$this->assertTrue( user_can( $user, Permissions::VIEW_SPLASH ) );
 	}
+
+	public function test_permissions_route_unauthorized_request() {
```

Comment:
Elsewhere we are using the convention of separating the "test case identifier" from the variant, which I think is quite nice. I'd suggest applying it here too.

```suggestion
	public function test_permissions_route__unauthorized_request() {
```